### DISCUSSION TITLE
Changed upload limits get not applied

### DISCUSSION THREAD

**Discussion by KnudH** [requarks/wiki#3504] [Error / Bug Report]
See all in #2221 

I dont know why that issue was closed, but the it is not fixed and i can reproduce in latest version (2.5.159). Im using docker-compose. For sure I changed config of reverse proxy.

### Actual behavior
See #2221

### Expected behavior
See #2221 

### Steps to reproduce the behavior
See #2221

### Host Info:
    OS: Debian 10 (wiki.js inside docker with docker-compose)
    Wiki.js version: 2.5.159
    Database engine: PostreSQL

**Comment by Smankusors** [comment]
where is the host info?

---

@NGPixel btw, Wiki.js also need to mention about DB max size too...

**Comment by KnudH** [comment]
> 
> 
> where is the host info?
> 
> @NGPixel btw, Wiki.js also need to mention about DB max size too...

I edited my first post and added that info.

**Comment by NGPixel** [comment]
The UI doesn't reflect the current setting (UI bug), but it is being applied correctly and larger files will be accepted just fine. Your issue is either at the reverse proxy level or the database which rejects your request because it is too large.

Do you have any errors in Wiki.js logs when uploading a file? What about the browser dev tools, what's the response from the server in the network tab?

**Comment by KnudH** [comment]
I didnt checked both, UI and the upload itself after each try. Upload is working fine now, so you are right, its only the UI what is wrong. 

//Edit: still a bug in 2.5.170

**Comment by NateScarlet** [comment]
UI is using a static text:
https://github.com/Requarks/wiki/blob/a7b9be3bf7fd508a18cf6a6d663092f6cff5a27b/client/components/editor/editor-modal-media.vue#L153

**Comment by farzadpanahi** [comment]
@KnudH  is this resolved for you? I still have the same issue. See #6440.
